tutorials/clinical/20_seeg.py

```diff
@@ -69,7 +69,8 @@
 
 # first we need a head to mri transform since the data is stored in "head"
 # coordinates, let's load the mri to head transform and invert it
-head_mri_t = mne.coreg.estimate_head_mri_t('sample_seeg', subjects_dir)
+this_subject_dir = op.join(misc_path, 'seeg')
+head_mri_t = mne.coreg.estimate_head_mri_t('sample_seeg', this_subject_dir)
```

Comment:
@larsoner not sure how this ever passed?

I went to: https://github.com/mne-tools/mne-python/commit/034ff3f03ae1ae01a6fe52e3e91e10682c977aff

and tried it locally and it didn't work.

Reply:
This should fix though. Lmk if this is incorrect.